app-admin/syslog-ng/syslog-ng-3.15.1.ebuild

```diff
@@ -0,0 +1,165 @@
+DEPEND="${RDEPEND}
+	sys-devel/flex
+	virtual/pkgconfig"
+
+DOCS=( AUTHORS NEWS.md CONTRIBUTING.md contrib/syslog-ng.conf.{HP-UX,RedHat,SunOS,doc}
+	contrib/syslog2ng "${FILESDIR}/${MY_PV_MM}/syslog-ng.conf.gentoo.hardened"
+	"${T}/syslog-ng.logrotate.hardened" "${FILESDIR}/README.hardened" )
+
+S=${WORKDIR}/${PN}-${MY_PV}
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	local f
+
+	use python && python_fix_shebang .
+
+	# remove bundled libs
+	rm -r lib/ivykis modules/afmongodb/mongo-c-driver modules/afamqp/rabbitmq-c || die
+
+	# drop scl modules requiring json
+	if use !json; then
+		sed -i -r '/cim|elasticsearch|ewmm|graylog2|loggly|logmatic/d' scl/Makefile.am || die
+	fi
+
+	# use gentoo default path
+	if use systemd; then
+		sed -e 's@/etc/syslog-ng.conf@/etc/syslog-ng/syslog-ng.conf@g;s@/var/run@/run@g' \
+			-i contrib/systemd/syslog-ng@default || die
+	fi
+
+	for f in "${FILESDIR}"/*logrotate*.in ; do
+		local bn=$(basename "${f}")
+
+		sed \
+			-e "$(usex systemd \
+				's/@GENTOO_RESTART@/systemctl kill -s HUP syslog-ng@default/' \
+				's:@GENTOO_RESTART@:/etc/init.d/syslog-ng reload:')" \
+			"${f}" > "${T}/${bn/.in/}" || die
+	done
+
+	default
+
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--disable-docs \
+		--disable-java \
+		--disable-java-modules \
+		--disable-riemann \
+		--enable-manpages \
+		--localstatedir=/var/lib/syslog-ng \
+		--sysconfdir=/etc/syslog-ng \
+		--with-embedded-crypto \
+		--with-ivykis=system \
+		--with-module-dir=/usr/$(get_libdir)/syslog-ng \
+		--with-pidfile-dir=/var/run \
+		--with-systemdsystemunitdir="$(systemd_get_systemunitdir)" \
+		$(use_enable amqp) \
+		$(usex amqp --with-librabbitmq-client=system --without-librabbitmq-client) \
+		$(use_enable caps linux-caps) \
+		$(use_enable dbi sql) \
+		$(use_enable geoip) \
+		$(use_enable http) \
+		$(use_enable ipv6) \
+		$(use_enable json) \
+		$(use_enable mongodb) \
+		$(usex mongodb --with-mongoc=system "--without-mongoc --disable-legacy-mongodb-options") \
+		$(use_enable pacct) \
+		$(use_enable python) \
+		$(use_enable redis) \
+		$(use_enable smtp) \
+		$(use_enable spoof-source) \
+		$(use_enable systemd) \
+		$(use_enable tcpd tcp-wrapper)
+}
+
+src_install() {
+	default
+
+	# Install default configuration
+	insinto /etc/default
+	doins contrib/systemd/syslog-ng@default
+
+	insinto /etc/syslog-ng
+	if use userland_BSD ; then
+		newins "${FILESDIR}/${MY_PV_MM}/syslog-ng.conf.gentoo.fbsd" syslog-ng.conf
+	else
+		newins "${FILESDIR}/${MY_PV_MM}/syslog-ng.conf.gentoo" syslog-ng.conf
+	fi
+
+	insinto /etc/logrotate.d
+	newins "${T}/syslog-ng.logrotate" syslog-ng
+
+	newinitd "${FILESDIR}/${MY_PV_MM}/syslog-ng.rc" syslog-ng
+	newconfd "${FILESDIR}/${MY_PV_MM}/syslog-ng.confd" syslog-ng
+	keepdir /etc/syslog-ng/patterndb.d /var/lib/syslog-ng
+	prune_libtool_files --modules
+
+	use python && python_optimize
+}
+
+pkg_postinst() {
+	# bug #355257
+	if ! has_version app-admin/logrotate ; then
+		echo
```

Comment:
Don't mix `echo` and `e*`, this will not make the Portage repeated elog messages more readable.